dbt_platform_helper/commands/environment.py

```diff
@@ -377,7 +378,6 @@ def generate_terraform(name):
 
 
 def _generate_copilot_environment_manifests(name, env_config):
```

Comment:
This function needs session arg adding

```suggestion
def _generate_copilot_environment_manifests(name, env_config, session):
```

Reply:
Thanks updated as suggested will run pytest locally next time :)